modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -276,6 +276,7 @@
     <string name="player_sleep_timer_start_failed">Unable to start timer. Please allow \'Alarms &amp; reminders\' permission.</string>
     <string name="player_sleep_when_episode_ends">Sleeping when episode ends</string>
     <string name="player_star" translatable="false">@string/star</string>
+    <string name="player_tab_bookmarks">Bookmarks</string>
```

Comment:
As we need Bookmarks in multiple places maybe we should either just have `bookmarks` or maybe link to a common string. 
```
<string name="player_tab_bookmarks" translatable="false">@string/bookmarks</string>
```

Reply:
Makes sense, updated in 5658c6f